### ISSUE TITLE
Download page

### ISSUE THREAD

**Issue by Turbotailz** [Turbotailz/LuckPermsWeb#28]
- [ ] List of available downloads
- [ ] "Not sure which version?" quiz
- [ ] Links to relevant install pages
- [ ] Other relevant links (e.g. Jenkins, Essentials?)

**Comment by Andre601** [comment]
I would agree with point 2 assuming you mean like a thing where you can select different options (e.g. Server/Proxy type and version, if it is Bungee or not, etc.) and the site recommends the right download.

**Comment by Turbotailz** [comment]
Yeah, basically just asks a few simple yes/no questions like "Are you running a network"? "What version of Minecraft?" etc.

Can prompt users to update their version if they say 1.8.0 or something.
